Tests/.editorconfig

```diff
@@ -214,6 +214,9 @@ dotnet_diagnostic.MA0096.severity = none
 # A class that implements IComparable<T> or IComparable should override comparison operators. Don't care about this.
 dotnet_diagnostic.MA0097.severity = none
 
+# Do not convert implicitly to DateTimeOffset. Fair point, but we need this in our tests
+dotnet_diagnostic.MA0132.severity = error
```

Comment:
Initially yes because I got the impression it was needed to have Rider report the violations, but I've just checked again and we don't. So removed it.